finngen/run_finngen_pipeline.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
```

Comment:
Do you think this script will take a while and does it print a lot to stdout? If it has no progress indicator and takes a long time, I think it may be useful to a log message with a timestamp before starting each script, just so we have an idea where we are at. But that's really only useful if it takes a long time. I made a small commit to inject those kinds of messages for the coloc step: https://github.com/thehyve/otg-data-loading/commit/52d1fc777b0435a32f1fe8a955e92d9bef49913c

Reply:
I am not entirely sure since I only tested with 1 file but I think it really shouldn't take a lot of time. It's just reformatting some input to output that can be handled by the pipeline. Also, the formatting of this 1 file was finished way faster then the `real` finemapping step.

Reply:
Ok, then adding log statements would probably just produce noise and not much use.